### DISCUSSION TITLE
How to connect a USB Android device to a Appium Server in Kubernetes via ADB

### DISCUSSION THREAD

**Discussion by uman3951** [appium/appium#20622] [Q&A]
I want to connect a USB Android real device to the Appium server which is running in the Container in Kubernetes environment

I deployed the Appium server (Appium 2.0) in Kubernetes and I connected my real Android device to my local machine via ADB. I want to connect my Android Device to the Appium Server which is running in the Kubernetes

**Comment by KazuCocoa** [comment]
I guess you would need to do like https://developer.android.com/tools/adb#wireless, or attach to adb server running on your machine from the Kubernetes env with `appium:remoteAdbHost` capability. I haven't tried out, thus asking such questions in https://discuss.appium.io/ would also help
